package.json

```diff
@@ -85,7 +86,8 @@
     "prettier": "^2.2.1",
     "sass": "^1.29.0",
     "sass-loader": "^8.0.2",
-    "typescript": "~4.1.5"
+    "typescript": "~4.1.5",
+    "webpack": "^4.1.0"
```

Comment:
any reason for adding `webpack`? 🤔

Reply:
@ChaituVR See https://github.com/snapshot-labs/snapshot/pull/380/files#r646190071